added culture and added curiosities

monikasarnecka opened the pull request.

LukKaszuba commented:
Super :) jedyna rzetelna! Aż mnie zmotywowałaś żeby to zrobić chociaż nie wiem czy będę miał kiedy… dzisiaj cały dzień temat Ukrainy :(

Dziękuję i pozdrawiam ll Thank you & Best Regards
~ Łukasz Kaszuba 

> Wiadomość napisana przez monikasarnecka ***@***.***> w dniu 24.02.2022, o godz. 20:43:
> 
> ﻿
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1
> 
> Commit Summary
> 
> 900409b added culture and added curiosities
> File Changes  (32 files)
> A bootstrap-4.1.3-dist/_DS_Store (0)
> A bootstrap-4.1.3-dist/css/_DS_Store (0)
> A bootstrap-4.1.3-dist/css/bootstrap-grid.css (1912)
> A bootstrap-4.1.3-dist/css/bootstrap-grid.css.map (1)
> A bootstrap-4.1.3-dist/css/bootstrap-grid.min.css (7)
> A bootstrap-4.1.3-dist/css/bootstrap-grid.min.css.map (1)
> A bootstrap-4.1.3-dist/css/bootstrap-reboot.css (331)
> A bootstrap-4.1.3-dist/css/bootstrap-reboot.css.map (1)
> A bootstrap-4.1.3-dist/css/bootstrap-reboot.min.css (8)
> A bootstrap-4.1.3-dist/css/bootstrap-reboot.min.css.map (1)
> A bootstrap-4.1.3-dist/css/bootstrap.css (9030)
> A bootstrap-4.1.3-dist/css/bootstrap.css.map (1)
> A bootstrap-4.1.3-dist/css/bootstrap.min.css (7)
> A bootstrap-4.1.3-dist/css/bootstrap.min.css.map (1)
> A bootstrap-4.1.3-dist/js/bootstrap.bundle.js (6461)
> A bootstrap-4.1.3-dist/js/bootstrap.bundle.js.map (1)
> A bootstrap-4.1.3-dist/js/bootstrap.bundle.min.js (7)
> A bootstrap-4.1.3-dist/js/bootstrap.bundle.min.js.map (1)
> A bootstrap-4.1.3-dist/js/bootstrap.js (3944)
> A bootstrap-4.1.3-dist/js/bootstrap.js.map (1)
> A bootstrap-4.1.3-dist/js/bootstrap.min.js (7)
> A bootstrap-4.1.3-dist/js/bootstrap.min.js.map (1)
> A img/colors.jpg (0)
> A img/dragon.jpg (0)
> A img/moon.jpg (0)
> A img/photothumb.db (0)
> A img/porcelain.jpg (0)
> A img/smok4.png (0)
> A img/tai chi.png (0)
> A img/tea.jpg (0)
> M index.html (175)
> M style.css (66)
> Patch Links:
> 
> https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1.patch
> https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1.diff
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android. 
> You are receiving this because you are subscribed to this thread.

monikasarnecka commented:
I tak w ten weekend chyba nie mamy zajęć z Grzegorzem więc chyba nie ma pośpiechu 🙂 ja zrobiłam żeby się czegoś nauczyć ale rozumiem że nie każdy ma na to czas 😉Temat: Re: [LukKaszuba/Zadanie-Domowe-Team-1] added culture and added curiosities (PR #1)Data: 2022-02-24 21:36Nadawca: "LukKaszuba" ***@***.***&gt;Adresat: "LukKaszuba/Zadanie-Domowe-Team-1" ***@***.***&gt;; DW: "monikasarnecka" ***@***.***&gt;; "Author" ***@***.***&gt;; 

Super :) jedyna rzetelna! Aż mnie zmotywowałaś żeby to zrobić chociaż nie wiem czy będę miał kiedy… dzisiaj cały dzień temat Ukrainy :(



Dziękuję i pozdrawiam ll Thank you &amp; Best Regards

~ Łukasz Kaszuba 



&gt; Wiadomość napisana przez monikasarnecka ***@***.***&gt; w dniu 24.02.2022, o godz. 20:43:

&gt; 

&gt; ﻿

&gt; You can view, comment on, or merge this pull request online at:

&gt; 

&gt;   https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1

&gt; 

&gt; Commit Summary

&gt; 

&gt; 900409b added culture and added curiosities

&gt; File Changes  (32 files)

&gt; A bootstrap-4.1.3-dist/_DS_Store (0)

&gt; A bootstrap-4.1.3-dist/css/_DS_Store (0)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.css (1912)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.css.map (1)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.min.css (7)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.min.css.map (1)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.css (331)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.css.map (1)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.min.css (8)

&gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.min.css.map (1)

&gt; A bootstrap-4.1.3-dist/css/bootstrap.css (9030)

&gt; A bootstrap-4.1.3-dist/css/bootstrap.css.map (1)

&gt; A bootstrap-4.1.3-dist/css/bootstrap.min.css (7)

&gt; A bootstrap-4.1.3-dist/css/bootstrap.min.css.map (1)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.js (6461)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.js.map (1)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.min.js (7)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.min.js.map (1)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.js (3944)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.js.map (1)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.min.js (7)

&gt; A bootstrap-4.1.3-dist/js/bootstrap.min.js.map (1)

&gt; A img/colors.jpg (0)

&gt; A img/dragon.jpg (0)

&gt; A img/moon.jpg (0)

&gt; A img/photothumb.db (0)

&gt; A img/porcelain.jpg (0)

&gt; A img/smok4.png (0)

&gt; A img/tai chi.png (0)

&gt; A img/tea.jpg (0)

&gt; M index.html (175)

&gt; M style.css (66)

&gt; Patch Links:

&gt; 

&gt; https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1.patch

&gt; https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1.diff

&gt; —

&gt; Reply to this email directly, view it on GitHub, or unsubscribe.

&gt; Triage notifications on the go with GitHub Mobile for iOS or Android. 

&gt; You are receiving this because you are subscribed to this thread.





—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

LukKaszuba commented:
To dobra wiadomość!! Nawet nie wiesz jak. Właśnie wziąłem urlop w przyszłym tygodniu bo już nie dawałem rady z pracą innymi zadaniami. 

Dziękuję i pozdrawiam ll Thank you & Best Regards
~ Łukasz Kaszuba 

> Wiadomość napisana przez monikasarnecka ***@***.***> w dniu 24.02.2022, o godz. 22:46:
> 
> ﻿
> I tak w ten weekend chyba nie mamy zajęć z Grzegorzem więc chyba nie ma pośpiechu 🙂 ja zrobiłam żeby się czegoś nauczyć ale rozumiem że nie każdy ma na to czas 😉Temat: Re: [LukKaszuba/Zadanie-Domowe-Team-1] added culture and added curiosities (PR #1)Data: 2022-02-24 21:36Nadawca: "LukKaszuba" ***@***.***&gt;Adresat: "LukKaszuba/Zadanie-Domowe-Team-1" ***@***.***&gt;; DW: "monikasarnecka" ***@***.***&gt;; "Author" ***@***.***&gt;; 
> 
> Super :) jedyna rzetelna! Aż mnie zmotywowałaś żeby to zrobić chociaż nie wiem czy będę miał kiedy… dzisiaj cały dzień temat Ukrainy :(
> 
> 
> 
> Dziękuję i pozdrawiam ll Thank you &amp; Best Regards
> 
> ~ Łukasz Kaszuba 
> 
> 
> 
> &gt; Wiadomość napisana przez monikasarnecka ***@***.***&gt; w dniu 24.02.2022, o godz. 20:43:
> 
> &gt; 
> 
> &gt; ﻿
> 
> &gt; You can view, comment on, or merge this pull request online at:
> 
> &gt; 
> 
> &gt; https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1
> 
> &gt; 
> 
> &gt; Commit Summary
> 
> &gt; 
> 
> &gt; 900409b added culture and added curiosities
> 
> &gt; File Changes (32 files)
> 
> &gt; A bootstrap-4.1.3-dist/_DS_Store (0)
> 
> &gt; A bootstrap-4.1.3-dist/css/_DS_Store (0)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.css (1912)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.css.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.min.css (7)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-grid.min.css.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.css (331)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.css.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.min.css (8)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap-reboot.min.css.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap.css (9030)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap.css.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap.min.css (7)
> 
> &gt; A bootstrap-4.1.3-dist/css/bootstrap.min.css.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.js (6461)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.js.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.min.js (7)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.bundle.min.js.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.js (3944)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.js.map (1)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.min.js (7)
> 
> &gt; A bootstrap-4.1.3-dist/js/bootstrap.min.js.map (1)
> 
> &gt; A img/colors.jpg (0)
> 
> &gt; A img/dragon.jpg (0)
> 
> &gt; A img/moon.jpg (0)
> 
> &gt; A img/photothumb.db (0)
> 
> &gt; A img/porcelain.jpg (0)
> 
> &gt; A img/smok4.png (0)
> 
> &gt; A img/tai chi.png (0)
> 
> &gt; A img/tea.jpg (0)
> 
> &gt; M index.html (175)
> 
> &gt; M style.css (66)
> 
> &gt; Patch Links:
> 
> &gt; 
> 
> &gt; https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1.patch
> 
> &gt; https://github.com/LukKaszuba/Zadanie-Domowe-Team-1/pull/1.diff
> 
> &gt; —
> 
> &gt; Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> &gt; Triage notifications on the go with GitHub Mobile for iOS or Android. 
> 
> &gt; You are receiving this because you are subscribed to this thread.
> 
> 
> 
> 
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
> 
> You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
> 
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android. 
> You are receiving this because you commented.
